bsconfig.json

```diff
@@ -18,6 +17,6 @@
     }
   ],
   "plugins": [
-    "@rokucommunity/bslint"
```

Comment:
temporarily removed until bslint alpha-35 release

Reply:
I just released bslint alpha-35, so you should be good-to-go now.